app/models/cart.rb

```diff
@@ -0,0 +1,21 @@
+class Cart < ApplicationRecord
+  has_many :line_items, dependent: :nullify
```

Comment:
This is our cart class that is a part of the larger checkout composition.

I considered not making this an AR model and instead just a way to better conceptualize and access the underlining line item data, but it felt a bit too cumbersome. We don't want rogue line items in out database without an owner but as we can see there really isn't much AR-related code on the cart.

I thought about just maintaining a reference of some sort to act as a foreign key to keep the line items grouped together in an identifiable way. Because of this I felt I would be fighting the more straightforward solution of having a foreign key reference back to separate table.

So although there is currently minimal usage of AR-related code for our Cart, it is by far the easiest idea to comprehend and extend going forward.